README.md

```diff
@@ -14,14 +14,14 @@ Download
 
 Download [the latest JAR][2] or grab via Gradle:
 ```groovy
-implementation 'com.squareup.picasso:picasso:2.71828'
+implementation 'com.squareup.picasso3:picasso:3.0'
```

Comment:
I would leave the README. You're just going to confuse people.

Reply:
Good call.  I confused this with the website, which won't be published until the next release.